genstudio-mlr-claims-app/src/genstudiopem/web-src/src/components/ClaimsChecker.tsx

```diff
@@ -11,25 +11,26 @@ governing permissions and limitations under the License.
 */
 
 import React from 'react';
-import { Flex, Heading, Text, View, Button, InlineAlert, Badge } from "@adobe/react-spectrum";
+import { Flex, Heading, Text, View, Button, InlineAlert, Badge, Divider } from "@adobe/react-spectrum";
 import Alert from "@spectrum-icons/workflow/Alert";
-import { claimStatus } from '../utils/claimsValidation';
+import { Violation } from '../utils/claimsValidation';
```

Comment:
Renaming because now we are not just doing claim check so it makes more sense to call it violation.